### PR TITLE
Fix APASS passband names

### LINKED ISSUE #439: Current `apass_dr9` implementation does not change passbands to the AAVSO standard
The `apass_dr9` implementation does not change the passbands present in the Vizier catalog to the AAVSO standard names. It should.

### PR DESCRIPTION
**Pull request by mwcraig**
This PR ensures that the APASS passband names are the AAVSO standard names.

Fixes #439

### CODE REVIEW

**Comment by mwcraig** [comment]
Made this a draft until I figure out what caused the test failure....

**Comment by mwcraig** [comment]
...and I found the issue. I made the smallest possible change to get the tests to pass and open #441 to note the underlying problem.

**Comment by mwcraig** [comment]
@JuanCab -- this is ready for review now

**Comment by mwcraig** [comment]
Just added a test that hits the line that was not covered.

File: stellarphot/core.py [hunk 1]
```diff
@@ -1017,6 +1032,21 @@ def apass_dr9(field_center, radius=1 * u.degree, clip_by_frame=False, padding=10
 
     # IAU says there is a space between the acronym and the coordinates.
     raw_catalog["id"] = [f"{designation_acronym} {coord}" for coord in coord_string]
+
+    # Translate the passbands to AAVSO standard names.
+    # No need to change B and V since those are already correct.
+    # Do this *after* initialization so that the original APASS band names
+    # are used for the tidy-ification operation.
+    raw_catalog.passband_map = PassbandMap(
```
Comment (JuanCab) [inline stellarphot/core.py:1040]: I assume Vizier only tracks these bandpasses, so you can hardcode this, right?

**Review by JuanCab — APPROVED** [review]
Looks good to me, and no typos as far as I can see.

File: stellarphot/core.py (hunk 1 above)
Comment (mwcraig) [reply stellarphot/core.py:1040]: Each catalog seems to be difference 🎉 🙄 but for APASS these are the names in Vizier. A new data release of APASS could conceivably change that, but this is explicitly DR9, so a new data release would mean adding a new class (which is fine I think).

**Pull request merged**